controllers/argocd/util_test.go

```diff
@@ -460,3 +461,54 @@ func TestDeleteRBACsForNamespace(t *testing.T) {
 	assert.NilError(t, err)
 	assert.DeepEqual(t, string(s.Data["namespaces"]), "testNamespace2")
 }
+
+func TestRemoveManagedByLabelFromNamespaces(t *testing.T) {
+	a := makeTestArgoCD()
+	r := makeTestReconciler(t)
+
+	ns := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{
+		Name: "testNamespace",
+		Labels: map[string]string{
+			common.ArgoCDManagedByLabel: a.Namespace,
+		}},
+	}
+
+	err := r.client.Create(context.TODO(), ns)
```

Comment:
Client method became public with Operator SDK upgrade, please check.

Reply:
ahhh thanks .. missed it for the test files!